pages/Nvidia/_index.md

```diff
@@ -85,38 +101,118 @@ env = __GLX_VENDOR_LIBRARY_NAME,nvidia
+solve certain issues in electron apps, and it will also allow for GPU decoding
+for videos on the web, with benefits including higher performance video playback
+and also higher power efficiency during video playback.
+
+Its install instructions are available in the README, however, a quick guide will
+be given here:
+
+- Install the package. On Arch, this is `libva-nvidia-driver` in the official
+  repos.
+
+- export this variable in your hyprland config:
+  ```sh
+  env = NVD_BACKEND,direct
+  ```
+  see [here](https://github.com/elFarto/nvidia-vaapi-driver?tab=readme-ov-file#upstream-regressions)
+  for more information on this environment variable.
 
-If you face problems with Discord windows not displaying or screen sharing not
+- Enable the services `nvidia-suspend.service`, `nvidia-hibernate.service` and
+`nvidia-resume.service`
+
+- Add `nvidia.NVreg_PreserveVideoMemoryAllocations=1` to your kernel parameters if
+you don't have it already. This will solve issues with corrupted desktop / videos
+after waking.
+
+- You can check the README to get it working for Firefox.
+
+## Other issues to look out for
+
+### Regarding environment variables
+
+- If you encounter crashes in Firefox, remove the line
+`env = GBM_BACKEND,nvidia-drm`.
+
+- If you face problems with Discord windows not displaying or screen sharing not
 working in Zoom, remove or comment the line
 `env = __GLX_VENDOR_LIBRARY_NAME,nvidia`.
 
-{{< /callout >}}
+### How to possibly get multi-monitor working with hybrid graphics
 
-Install `qt5-wayland`, `qt5ct` and `libva`. Additionally
-`libva-nvidia-driver-git` (AUR) to fix crashes in some Electron-based
-applications, such as Unity Hub.
+On a hybrid graphics device (a laptop with
+both an Intel and an Nvidia GPU), you will need to remove the `optimus-manager`
+package if installed (disabling the service does not work). You also need to
+change your BIOS settings from hybrid graphics to discrete graphics.
 
-Reboot your computer
+### Fixing flickering in electron apps
 
-Launch Hyprland.
+This flickering is likely caused by these apps running in XWayland.
+To fix the flickering, try running the apps with native Wayland instead.
 
-It _should_ work now.
+For discord, you can try the [vesktop app](https://github.com/Vencord/Vesktop).
+It has a multitude of install options for many distros in their README.
+
+After installing, you can then enable the wayland backend with this command:
+`echo "--enable-features=UseOzonePlatform --ozone-platform-hint=auto" > ~/.config/vesktop-flags.conf`
+
+This will add the parameters required to run vesktop with its wayland backend.
+
+On earlier nvidia driver versions, including 535, you may have to also include
+the `--disable-gpu` and `--disable-gpu-sandbox` flags, but as the names suggest,
+you will lose hardware acceleration for vesktop.
+
+In other cases like vscodium, you can add the same flags to their respective
+flags files. In vscodium's case, this would be `~/.config/codium-flags.conf`
```

Comment:
```suggestion
For Discord, you can try the [Vesktop app](https://github.com/Vencord/Vesktop).
It has a multitude of install options for many distros in their README.

After installing, you can then enable the Wayland backend with this command:
`echo "--enable-features=UseOzonePlatform --ozone-platform-hint=auto" > ~/.config/vesktop-flags.conf`

This will add the parameters required to run Vesktop with its Wayland backend.

On earlier Nvidia driver versions, including 535, you may have to also include
the `--disable-gpu` and `--disable-gpu-sandbox` flags, but, as the names suggests,
you will lose hardware acceleration for Vesktop.

In other cases like VSCodium, you can add the same flags to their respective
flags files. In VSCodium's case, this would be `~/.config/codium-flags.conf`
```